src/man/sssd-passkey.5.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DocBook V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<reference>
+<title>SSSD Manual pages</title>
+<refentry>
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="include/upstream.xml" />
+
+    <refmeta>
+        <refentrytitle>sssd-passkey</refentrytitle>
+        <manvolnum>5</manvolnum>
+        <refmiscinfo class="manual">File Formats and Conventions</refmiscinfo>
+    </refmeta>
+
+    <refnamediv id='name'>
+        <refname>sssd-passkey</refname>
+        <refpurpose>SSSD passkey options</refpurpose>
+    </refnamediv>
+
+    <refsect1 id='description'>
+        <title>DESCRIPTION</title>
+        <para>
+            This manual page describes the specifics for configuration of
+            passkey for
+            <citerefentry>
+                <refentrytitle>sssd</refentrytitle>
+                <manvolnum>8</manvolnum>
+            </citerefentry>.
+            Refer to the <quote>FILE FORMAT</quote> section of the
+            <citerefentry>
+                <refentrytitle>sssd.conf</refentrytitle>
+                <manvolnum>5</manvolnum>
+            </citerefentry> manual page for detailed syntax information.</para>
+    </refsect1>
+
+    <refsect1 id='configuration-options'>
+        <title>CONFIGURATION OPTIONS</title>
+        <para>
+            <variablelist>
+                <varlistentry>
+                    <term>user_verification (boolean)</term>
+                    <listitem>
+                        <para>
+                            Enable or disable the user verification (i.e. PIN,
+                            fingerprint) during authentication.
+                        </para>
+                        <para>
+                            Three different actors come into play when deciding
+                            whether to request user verification: LDAP server,
+                            <citerefentry><refentrytitle>sssd.conf
+                            </refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                            option and the passkey device itself.
+                        </para>
+                        <para>
+                            The configuration option from <filename>sssd.conf
+                            </filename> has higher priority than the LDAP
+                            server and will override them when conflicts occur.
```

Comment:
I haven't tested local passkey auth but at least in the kerberos case this should not be true. See https://github.com/SSSD/sssd/blob/master/src/responder/pam/pamsrv_passkey.c#L175-L180

Reply:
Didn't we agree in the design phase that `sssd.conf` takes precedence over IPA settings?

Reply:
Hi,

I think at least in the Kerberos case it does not make sense to overwrite what we got form the KDC because iiuc the reply from the device which we send back to the KDC will contain information if user verification was used or not. If the KDC requires user verification and we would allow locally a device without a PIN the reply would be rejected by the KDC.

Maybe, it would be easier to say that with the IPA provider the server side settings will be used (since IPA is currently the only provider which supports server side settings) and for every other provider the settings from sssd.conf are used?

bye,
Sumit

Reply:
> Maybe, it would be easier to say that with the IPA provider the server side settings will be used (since IPA is currently the only provider which supports server side settings) and for every other provider the settings from sssd.conf are used?

I think this is the best solution, as it enables us to have a consistent approach and we do not confuse users/sysadmins.

Reply:
Ok, I'm fine with this.

Reply:
I'm also okay with this.

Reply:
I have updated the text to express what was discussed in this issue